File: src/main/java/edu/eci/arsw/warGame/model/Player.java
package edu.eci.arsw.warGame.model;

public class Player {
    String usuario;
    String paisInicial;
    String colorPlayer;

    public String getPaisInicial() {
        return paisInicial;
    }

    public String getColorPlayer() {
        return colorPlayer;
    }

    public String getUsuario() {
        return usuario;
    }

    public void setPaisInicial(String paisInicial) {
        this.paisInicial = paisInicial;
    }

    public void setUsuario(String usuario) {this.usuario = usuario;}

    public void setColorPlayer(String colorPlayer) {
        this.colorPlayer = colorPlayer;
    }
}

File: src/main/resources/static/js/Player.js


class Player {
    constructor(usuario, paisInicial, ColorPlayer){
        this.usuario = usuario;
        this.paisInicial =paisInicial;
        this.colorPlayer = ColorPlayer;
    }

    get getUser(){
        return this.usuario;
    }
    get getPaisInicial(){
        return this.paisInicial;
    }
    get colorJugador(){
        return this.ColorPlayer;
    }


}